sketchlib/Tween.js

```diff
@@ -0,0 +1,76 @@
+import { Point } from "../pga2d/objects.js";
+import { clamp } from "./clamp.js";
+
+function lerp_number(a, b, t) {
```

Comment:
I'm pretty sure I have this elsewhere in `sketchlib`... make a common file for it.

Reply:
Apparently not... for now let's leave it alone until it's needed elsewhere.